### PR TITLE
badger added and working

### PR DESCRIPTION
**Pull request by dpet**

### CODE REVIEW

File: src/assets/js/badger.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      canvas: document.getElementById("badger-canvas"),
+      autoplay: true,
+      artboard: "Artboard",
+      stateMachines: "State Machine 1",
+  });
+  pass_mousemove_events()
+
+}
+
+export function lazer_eyes(){
```
Comment (uncomputable) [inline src/assets/js/badger.js:15]: typo: "lazer" → "laser"

File: src/assets/animations/simplicity_logo.svg
Comment (uncomputable) [inline src/assets/animations/simplicity_logo.svg:1]: This file already exists in [src/assets/images](https://github.com/uncomputable/simfony-webide/tree/85c24314d02b935bffb73725be9d5306409168bd/src/assets/images). Please remove the logo from the animations folder.

**Comment by uncomputable** [comment]
> Error: Failed to invoke action displaySetContentScale: frame state invalid

Switching to mobile causes an error that seems to disable the ducking animation.

File: src/assets/js/badger.js (hunk 1 above)
Comment (dpet) [reply src/assets/js/badger.js]: fixed

File: src/assets/animations/simplicity_logo.svg
Comment (dpet) [reply src/assets/animations/simplicity_logo.svg:1]: removed

**Comment by dpet** [comment]
I was unable to duplicate the firefox mobile error. In Chrome, when it emulates a mobile device, the cursor does not have a hover state because you can only press or not press on a tablet or phone. Clicking on the screen activates the hover state and pressing somewhere else deactivates it. I added error catching so errors in the js don't stop the subsequent rust functions from running.

**Pull request merged**